docs/mp/aot.adoc

```diff
@@ -31,10 +31,22 @@ application pom as a parent of your module, you can use the following steps to
 build a native image from your application:
 
 1. Create an environment variable `GRAALVM_HOME` pointing to your installation of
-    GraalVM with `native-image` installed
+    GraalVM, or use a GraalVM JDK to build your project
+2. Add configuration to initialize your packages at build time (see below)
 2. Run Maven command `mvn clean package -Pnative-image`
 3. Execute the native executable created in `target` directory of your project
 
+== Initialize at build time
+Due to restrictions of our current implementation of integration with native image for CDI,
+we must initialize a subset of types at build time.
+As a result, all beans created within CDI need to be also initialized at build time.
```

Comment:
Too much internal details.
How about the following instead:

```suggestion
Helidon MP requires the types of all beans created within CDI to be initialized at build time.
```

Reply:
Reworded